tests/lib/nested.sh

```diff
@@ -117,6 +129,10 @@ start_nested_core_vm_unit(){
     if [ "${ENABLE_TPM}" = "true" ]; then
         if ! snap list swtpm-mvo > /dev/null; then
             snap install swtpm-mvo --beta
+            retry=10
+            while ! test -S /var/snap/swtpm-mvo/current/swtpm-sock; do
+                sleep 1
```

Comment:
Sorry, I'm a Muppet - this loop of course should decrease retry and eventually stop instead of looping forever (spread will kill it eventually of course but still a bit silly). :+100: on the idea to share code from snapd, just the fact that there is no "retry" helper is a major hindrance. Oh well - one step at a time :)

Reply:
Apparently /me is more of a Muppet for not noticing /o\

Reply:
I fixed this now